src/test/link-rewriter.test.js

```diff
@@ -19,14 +19,14 @@ const OrgUnit = {
 test.serial('rewriteLinks updates module link to topic', async t => {
 	mockFs({
 		'/testdata/module1/index.html': '<html><body><a href="module1/topic1.md">Topic 1</a></body></html>',
-		'/testdata/module1/topic1.html': '<html><body><p>Content</p></body></html>'
+		'/testdata/module1/topic1.md': '<html><body><p>Content</p></body></html>'
```

Comment:
Doesn't this content break when piped through `marked`?

Reply:
I can add a test to verify it leaves html unmodified.   I agree it would be clearer for the .md file to actually contain markdown though  :)